static/app/components/events/eventReplay/index.tsx

```diff
@@ -1,63 +1,50 @@
-import {Fragment, lazy} from 'react';
-import ReactLazyLoad from 'react-lazyload';
-import styled from '@emotion/styled';
+import {lazy} from 'react';
 
-import {LinkButton} from 'sentry/components/button';
-import NegativeSpaceContainer from 'sentry/components/container/negativeSpaceContainer';
 import ErrorBoundary from 'sentry/components/errorBoundary';
-import {REPLAY_LOADING_HEIGHT} from 'sentry/components/events/eventReplay/constants';
-import {EventReplaySection} from 'sentry/components/events/eventReplay/eventReplaySection';
+import {ReplayClipSection} from 'sentry/components/events/eventReplay/replayClipSection';
 import LazyLoad from 'sentry/components/lazyLoad';
-import LoadingIndicator from 'sentry/components/loadingIndicator';
-import {ReplayGroupContextProvider} from 'sentry/components/replays/replayGroupContext';
 import {replayBackendPlatforms} from 'sentry/data/platformCategories';
-import {t, tct} from 'sentry/locale';
-import {space} from 'sentry/styles/space';
 import type {Event} from 'sentry/types/event';
 import type {Group} from 'sentry/types/group';
-import {getAnalyticsDataForEvent, getAnalyticsDataForGroup} from 'sentry/utils/events';
-import useReplayCountForIssues from 'sentry/utils/replayCount/useReplayCountForIssues';
 import {getReplayIdFromEvent} from 'sentry/utils/replays/getReplayIdFromEvent';
 import {useHasOrganizationSentAnyReplayEvents} from 'sentry/utils/replays/hooks/useReplayOnboarding';
 import {projectCanUpsellReplay} from 'sentry/utils/replays/projectSupportsReplay';
 import useOrganization from 'sentry/utils/useOrganization';
 import useProjectFromSlug from 'sentry/utils/useProjectFromSlug';
-import useRouter from 'sentry/utils/useRouter';
-import {normalizeUrl} from 'sentry/utils/withDomainRequired';
 
-type Props = {
+interface Props {
   event: Event;
   projectSlug: string;
   group?: Group;
-};
-
-export const REPLAY_CLIP_OFFSETS = {
-  durationAfterMs: 5_000,
-  durationBeforeMs: 5_000,
-};
+}
 
 const ReplayOnboardingPanel = lazy(() => import('./replayInlineOnboardingPanel'));
-const ReplayClipPreview = lazy(() => import('./replayClipPreview'));
 
-function EventReplayContent({
-  event,
-  group,
-  replayId,
-}: Props & {replayId: undefined | string}) {
+export default function EventReplay({event, group, projectSlug}: Props) {
   const organization = useOrganization();
-  const {hasOrgSentReplays, fetching} = useHasOrganizationSentAnyReplayEvents();
-  const router = useRouter();
-  const {getReplayCountForIssue} = useReplayCountForIssues();
+  const hasReplaysFeature = organization.features.includes('session-replay');
+
+  const {hasOrgSentReplays, fetching: fetchingHasSentReplays} =
+    useHasOrganizationSentAnyReplayEvents();
+  const project = useProjectFromSlug({organization, projectSlug});
+
+  const replayId = getReplayIdFromEvent(event);
 
-  if (fetching) {
-    return null;
+  if (!hasReplaysFeature || fetchingHasSentReplays) {
```

Comment:
what happens if we return `false` here? is it the same as returning `null`?

Reply:
Meant to be null. Caught me copy paste again! 

Null would be better.